src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp

```diff
@@ -73,8 +73,8 @@ void TargetedMovementGeneratorMedium<T, D>::_setTargetLocation(T* owner, bool up
             //   doing a "dance" while fighting
             if (owner->IsPet() && i_target->GetTypeId() == TYPEID_PLAYER)
             {
-                dist = i_target->GetCombatReach();
-                size = i_target->GetCombatReach() - i_target->GetObjectSize();
+                dist = 1.0f; //i_target->GetCombatReach();
```

Comment:
> please don't add comments that don't give any additional info then

My original commit https://github.com/TrinityCore/TrinityCore/commit/d1e6b603394357f3772e81779ff2741c6fdb6e8f had these comments:

```C++
dist = 1.0f; //i_target->GetCombatReach();
size = 1.0f; //i_target->GetCombatReach() - i_target->GetObjectSize();
```
I was trying to normalize the values because all pets should have the same reach / size.